cli_tools/common/image/importer/request.go

```diff
@@ -103,6 +103,9 @@ type ImageImportRequest struct {
 	Timeout               time.Duration `name:"timeout" validate:"required"`
 	UefiCompatible        bool
 	Zone                  string `name:"zone" validate:"required"`
+	// FormatsWithParallelInflation contains the file formats (eg: vmdk, vpc) that should be inflated
+	// in parallel with the PD API.
+	FormatsWithParallelInflation map[string]bool
```

Comment:
Naming for this map is inconsistent with shadowInflater.

Reply:
Switching to a quick implementation that disables the PD API